en/deploy-on-alibaba-cloud.md

```diff
@@ -334,34 +421,6 @@ You can customize this script. For example, you can remove the `module "bastion"
 >
 > You can copy the `deploy/aliyun` directory. But you cannot copy a directory on which the `terraform apply` operation is currently performed. In this case, it is recommended to clone the repository again and then copy it.
 
-## Destroy
```

Comment:
Please keep this section here, no need to move to the front.

Reply:
This is to keep it consistent with the [Chinese doc](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/deploy-on-alibaba-cloud.md). Please confirm whether we should move it.